services/storage/src/simcore_service_storage/handlers_simcore_s3.py

```diff
@@ -40,6 +40,8 @@
 
 @routes.post(f"/{api_vtag}/simcore-s3:access", name="get_or_create_temporary_s3_access")
 async def get_or_create_temporary_s3_access(request: web.Request) -> web.Response:
+    # NOTE: the name of the method is not accurate, these are not temporary at all
```

Comment:
you are right, that is a left over from where we were thinking of using STS to create temporary S3 access. I will create an issue cause it is linked with the computational backend.